docs/index.rst

```diff
@@ -60,20 +60,31 @@ Table of Contents
              :target: examples/carvallo-whipple.html
 
              Carvallo-Whipple bicycle model.
-      * - .. figure:: examples/kane-levinson-1985.png
+      * - .. figure:: examples/astrobee.png
```

Comment:
This wasn't showing up in my local build, I don't think you have it defined correctly.